ts/main.ts

```diff
@@ -0,0 +1,99 @@
+import { fetchTeams, Team } from './data';
+
+document.addEventListener('DOMContentLoaded', () => {
+  const teamsTab = document.getElementById('teams-tab') as HTMLButtonElement;
```

Comment:
When you change these back from modules to normal script tags you can remove this import. Also you do not need to wrap everything in a 'DOMContentLoaded' event listener.